src/components/Onboarding/Onboarding.scss

```diff
@@ -6,26 +6,52 @@
   height: 100vh;
   display: flex;
   justify-content: center;
-  align-items: center;
+  align-items: flex-start;
   background-color: #e7f0f0;
-  padding: 2rem 0 2rem 0;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+}
 
-  @at-root #{&}__content {
-    padding: 1rem;
-    max-width: 400px;
-    width: 100%;
-    background-color: #fff9f5 !important;
-  }
+.ai-onboarding .ai-onboarding-content {
+  max-width: 400px;
```

Comment:
I really happy with your work but, I think the app would be mobile-first, but supportable in all screens/ widths, what do you think?